src/registrar/models/domain_application.py

```diff
@@ -633,11 +655,13 @@ def submit(self):
         self.submission_date = timezone.now().date()
         self.save()
 
-        self._send_status_update_email(
-            "submission confirmation",
-            "emails/submission_confirmation.txt",
-            "emails/submission_confirmation_subject.txt",
-        )
+        # Limit email notifications for this transition to the first time the request transitions to this status
+        if not self.has_previously_had_a_status_of("submitted"):
```

Comment:
```suggestion
        if not self.has_previously_had_a_status_of(DomainApplication.ApplicationStatus.SUBMITTED):
```
(Optional) May need some slight tweaking, but why not use the constant for this field and the others?